ui/pages/swaps/build-quote/build-quote.stories.js

```diff
@@ -128,77 +125,34 @@ const tokensToSearch = tokens.map((token) => ({
 export default {
   title: 'Pages/Swaps/BuildQuote',
   id: __filename,
+  argTypes: {
+    ethBalance: {
+      control: { type: 'text' },
+    },
+    selectedAccountAddress: {
+      control: { type: 'text' },
+    },
+    selectedFromTokenIsNull: {
+      control: { type: 'boolean' },
+    },
+    loading: {
+      control: { type: 'boolean' },
+    },
+    shuffledTokensList: { control: 'object' },
+  },
+  args: {
+    ethBalance: '0x8',
+    selectedAccountAddress: 'selectedAccountAddress',
+    selectedFromTokenIsNull: false,
+    loading: false,
+    shuffledTokensList: [],
```

Comment:
@georgewrmarshall what should be the args for `shuffledTokensList` array

Reply:
The instance I can see is that it should be using the `shuffle` function from lodash? But I think it can be left blank for now.